src/collective/purgebyid/events.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from collective.purgebyid import logger
 from collective.purgebyid.api import getInvolvedObjs
 from collective.purgebyid.api import markInvolvedObjs
```

Comment:
I would use `mark_involved_objects` and not the old style.

Reply:
changed.